ci-operator/step-registry/workers-rhel/repos/workers-rhel-repos-commands.sh

```diff
@@ -72,6 +72,13 @@ cat > prep.yaml <<-'EOF'
       src: "rhel-{{ major_platform_version }}-server-rpms.repo.j2"
       dest: "/etc/yum.repos.d/rhel-{{ major_platform_version }}-server-rpms.repo"
 
+  - name: List current enabled repos and also make repo metadata cache in advance
+    shell: yum repolist
```

Comment:
Maybe we can use `{{ ansible_pkg_mgr }}` to replace `yum` here?

Reply:
But that need to enable `gather_facts` on https://github.com/openshift/release/blob/6e13cfc244530d431dcea1b3e433d1490cfc385d/ci-operator/step-registry/workers-rhel/repos/workers-rhel-repos-commands.sh#L36

Reply:
yes, it's ok for me to update that, we usually only running scaleup for 2 or 3 RHEl workers, gather_facts won't take too much time.